### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by arthurborgesdev**
# In this pull request I implemented:

- Calculator class component to call all the CalculatorButton components passing the button text as props
- CalculatorButton class component to render a calculator button with its respective text from props
- Styling to assemble the buttons towards a calculator design
- Add calculator screenshot to README
- Fix stylelint errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (DeliceLydia) [inline src/App.js:7]: - [ ] The `App.js` component should be class-based please work on that

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,8 @@
...
-    transform: rotate(0deg);
-  }
-
-  to {
-    transform: rotate(360deg);
-  }
+  align-items: center;
+  text-align: center;
+  min-height: 100vh;
 }
```
Comment (DeliceLydia) [inline src/App.css:8]: - [ ] Please delete all the boilerplate(texts, images, styles) will not need in your application

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @arthurborgesdev👏👏

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Design is looking okay now but you can do more to improve it to look exactly like the design

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 2 above)
Comment (arthurborgesdev) [reply src/App.css:8]: @DeliceLydia, the content in the App.css is not boilerplate. They are styles that I added to make the calculator to be rendered at the center of the screen.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations👏👏

## Hi @arthurborgesdev

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 2 above)
Comment (DeliceLydia) [reply src/App.css:8]: @arthurborgesdev I just highlighted because I was not able to highlight the boilerplate but I can see you did it.
Good job!

**Pull request merged**